docs/api.html

```diff
@@ -22,8 +22,9 @@
     window.addEventListener('load', function(){
 
       var cld = cloudinary.Cloudinary.new({ cloud_name: 'demo' });
+      var options = location.search.split('testUrlWithGet=')[1] ? {testUrlWithGet : true} : {};
```

Comment:
@maoznir, on second thought, I would change the queryParam to - testUrlWith=get/head

Reply:
Ok

Reply:
done